bin/node-sass

```diff
@@ -161,11 +163,17 @@ function getEmitter() {
   });
 
   emitter.on('log', function(data) {
-    console.log(data);
+    // If we need to outout large size buffer,
+    // keep waiting until buffer outputted completely.
+    asyncWaiting = true;
+
+    // In nodejs <= 7.1.0, console.log cannot write output larger than 8275 bytes data.
+    // So that, we output partial strings manually with some asynchronous way.
+    stdout(data);
   });
 
   emitter.on('done', function() {
-    if (!options.watch && !options.directory) {
+    if (!options.watch && !options.directory && !asyncWaiting) {
```

Comment:
I don't think this is right. I believe we always want to exit when emitter.log is called. This is what happens when you call the node-sass cli. If I understand correctly all we need to do is

```js
emitter.on('log', stdout.write);
```

Am I missing something?